### PR TITLE
DOC-4274: Document new property to disable activity link previews

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

File: docs/Configuration.rst [hunk 1]
```diff
@@ -96,6 +96,9 @@ Configuration
        Instructions on how to enable/disable an activity type from
        posting on streams.
 
+    -  :ref:`Enabling/Disabling Activity link preview <Configuration.ActivityLinkPreview>`
+       Instructions on how to enable/disable link preview in activity.
```
Comment (thomasdelhomenie) [inline docs/Configuration.rst]: I would shorten it to "How to enable/disable ...", like below.

File: docs/Configuration.rst [hunk 2]
```diff
@@ -2828,7 +2831,26 @@ To disable the activity type, simply set the parameter to ``false``.
...
+by defining the parameter ``exo.activity.link.preview.enabled`` 
+in :ref:`exo.properties <Configuration.ConfigurationOverview>` file.
+
+.. code::
+
+     #Enable/Disable Activity link preview
+     exo.activity.link.preview.enabled=true
+     
+By default, the parameter is set to true i.e. the activities link preview
+is enabled.
```
Comment (thomasdelhomenie) [inline docs/Configuration.rst:2856]: Shouldn't we add screenshots to make it clear what preview is, and what is the result when it is disabled ?

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/Configuration.rst (hunk 2 above)
Comment (mzorai) [reply docs/Configuration.rst:2856]: ok, i will apply the patch to make screenshots

**Comment by mzorai** [comment]
Content of the PR is viewable through this link https://docs-preprod.exoplatform.org/en/preproduction/Configuration.html#configuration-activitylinkpreview

File: docs/Configuration.rst [hunk 3]
```diff
@@ -2828,7 +2831,31 @@ To disable the activity type, simply set the parameter to ``false``.
...
+
+If you disable activities link preview i.e. set the variable 
+``exo.activity.link.preview.enabled`` to false, links will appears like that on the platform:
```
Comment (thomasdelhomenie) [inline docs/Configuration.rst]:
```suggestion
``exo.activity.link.preview.enabled`` to false, activities with links are rendered with the link URL only:
```

File: docs/Configuration.rst [hunk 4]
```diff
@@ -2828,7 +2831,31 @@ To disable the activity type, simply set the parameter to ``false``.
...
 			   connected together.
+		
+.. _Configuration.ActivityLinkPreview:		
+		
 			    
+==========================================
+Enabling/Disabling activity link preview
+==========================================
+
+eXo Platform allows you to enable or disable the link preview in an activity
```
Comment (thomasdelhomenie) [inline docs/Configuration.rst]: We should add an little introduction to describe what is link preview IMO (maybe there is already a documentation page for that ?), something like : "when posting an activity with a link attached, the activity display a preview for this link by fetching the title, an excerpt and a thumbnail of the linked resource", followed by a screenshot showing an example (with the same link than in the other screenshot to show the difference). WDYT ?

File: docs/Configuration.rst [hunk 5]
```diff
@@ -2828,7 +2831,31 @@ To disable the activity type, simply set the parameter to ``false``.
...
+Enabling/Disabling activity link preview
+==========================================
+
+eXo Platform allows you to enable or disable the link preview in an activity 
+by defining the parameter ``exo.activity.link.preview.enabled`` 
+in :ref:`exo.properties <Configuration.ConfigurationOverview>` file.
+
+.. code::
+
+     #Enable/Disable Activity link preview
```
Comment (thomasdelhomenie) [inline docs/Configuration.rst]: We can remove this comment.

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/Configuration.rst (hunk 4 above)
Comment (mzorai) [reply docs/Configuration.rst]: Yep I like the idea, I will update this part

File: docs/Configuration.rst (hunk 5 above)
Comment (mzorai) [reply docs/Configuration.rst]: ok

File: docs/Configuration.rst (hunk 4 above)
Comment (mzorai) [reply docs/Configuration.rst]: Updated

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**